2025-03-03 14:35:58.627215-05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create global temporary table HTE_client_table(rn_ integer not null, id bigint, apellido varchar(255), email varchar(255), nombre varchar(255), password varchar(255), telefono varchar(255), [*]user varchar(255), primary key (rn_)) TRANSACTIONAL"; expected "identifier"; SQL statement:
create global temporary table HTE_client_table(rn_ integer not null, id bigint, apellido varchar(255), email varchar(255), nombre varchar(255), password varchar(255), telefono varchar(255), user varchar(255), primary key (rn_)) TRANSACTIONAL [42001-232]
2025-03-03 14:35:59.126826-05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table client_table (\000d\000a        id bigint not null,\000d\000a        apellido varchar(255),\000d\000a        email varchar(255),\000d\000a        nombre varchar(255),\000d\000a        password varchar(255),\000d\000a        telefono varchar(255),\000d\000a        [*]user varchar(255),\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table client_table (
        id bigint not null,
        apellido varchar(255),
        email varchar(255),
        nombre varchar(255),
        password varchar(255),
        telefono varchar(255),
        user varchar(255),
        primary key (id)
    ) [42001-232]
2025-03-03 14:35:59.160019-05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CLIENT_TABLE" not found; SQL statement:

    alter table if exists pets_table 
       add constraint FKevt3v0i33pqi7reiembnl5j6o 
       foreign key (cliente_id) 
       references client [42102-232]
2025-03-03 14:36:03.032889-05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CLIENT_TABLE" not found; SQL statement:
insert into client_table (apellido,email,nombre,password,telefono,user,id) values (?,?,?,?,?,?,?) [42102-232]
2025-03-03 14:41:05.345844-05:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM CLIENT_TABLE CLIENT_TABLE [*]CLIENT_TABLE"; SQL statement:
SELECT * FROM CLIENT_TABLE CLIENT_TABLE CLIENT_TABLE [42000-232]
2025-03-03 14:43:13.023024-05:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CLIENTE" not found; SQL statement:
SELECT * FROM cliente WHERE LOWER(nombre) LIKE '%m%' [42102-232]
2025-03-03 20:29:21.913219-05:00 database: close
org.h2.message.DbException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.close(Store.java:374)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1292)
	at org.h2.engine.Database.closeImpl(Database.java:1254)
	at org.h2.engine.Database.close(Database.java:1173)
	at org.h2.engine.Database.onShutdown(Database.java:1158)
	at org.h2.engine.OnExitDatabaseCloser.onShutdown(OnExitDatabaseCloser.java:85)
	at org.h2.engine.OnExitDatabaseCloser.run(OnExitDatabaseCloser.java:114)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 8 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:926)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:463)
	at org.h2.mvstore.RandomAccessStore.compactStore(RandomAccessStore.java:443)
	at org.h2.mvstore.FileStore.compactStore(FileStore.java:890)
	at org.h2.mvstore.FileStore.stop(FileStore.java:275)
	at org.h2.mvstore.MVStore.closeStore(MVStore.java:693)
	at org.h2.mvstore.MVStore.close(MVStore.java:643)
	at org.h2.mvstore.db.Store.close(Store.java:364)
	... 6 more
Caused by: java.lang.AssertionError
	at org.h2.mvstore.RandomAccessStore.moveChunkInside(RandomAccessStore.java:630)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:569)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:481)
	at org.h2.mvstore.RandomAccessStore.lambda$compactMoveChunks$1(RandomAccessStore.java:468)
	at org.h2.mvstore.FileStore.executeFileStoreOperation(FileStore.java:1893)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:922)
	... 13 more
2025-03-03 20:33:04.137797-05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1423)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:463)
	at com.zaxxer.hikari.pool.HikariProxyConnection.getAutoCommit(HikariProxyConnection.java)
	at org.hibernate.resource.transaction.backend.jdbc.internal.DdlTransactionIsolatorNonJtaImpl.getIsolatedConnection(DdlTransactionIsolatorNonJtaImpl.java:49)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.getIsolatedConnection(GenerationTargetToDatabase.java:60)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.jdbcStatement(GenerationTargetToDatabase.java:112)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:79)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl$DelayedDropActionImpl.perform(SchemaDropperImpl.java:628)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryClosing(SessionFactoryObserverForSchemaExport.java:48)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryClosing(SessionFactoryObserverChain.java:48)
	at org.hibernate.internal.SessionFactoryImpl.close(SessionFactoryImpl.java:928)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:529)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:750)
	at jdk.proxy2/jdk.proxy2.$Proxy110.close(Unknown Source)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.destroy(AbstractEntityManagerFactoryBean.java:662)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:211)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:735)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:692)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:1401)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:651)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:1394)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1219)
	at org.springframework.context.support.AbstractApplicationContext.doClose(AbstractApplicationContext.java:1180)
	at org.springframework.context.support.AbstractApplicationContext.close(AbstractApplicationContext.java:1126)
	at org.springframework.boot.SpringApplicationShutdownHook.closeAndWait(SpringApplicationShutdownHook.java:147)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at org.springframework.boot.SpringApplicationShutdownHook.run(SpringApplicationShutdownHook.java:116)
	at java.base/java.lang.Thread.run(Thread.java:1575)
2025-03-03 20:33:04.145641-05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1423)
	at org.h2.jdbc.JdbcConnection.clearWarnings(JdbcConnection.java:658)
	at com.zaxxer.hikari.pool.ProxyConnection.close(ProxyConnection.java:258)
	at org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.closeConnection(DatasourceConnectionProviderImpl.java:131)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.releaseConnection(JdbcEnvironmentInitiator.java:472)
	at org.hibernate.resource.transaction.backend.jdbc.internal.DdlTransactionIsolatorNonJtaImpl.release(DdlTransactionIsolatorNonJtaImpl.java:106)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.release(GenerationTargetToDatabase.java:134)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl$DelayedDropActionImpl.perform(SchemaDropperImpl.java:639)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryClosing(SessionFactoryObserverForSchemaExport.java:48)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryClosing(SessionFactoryObserverChain.java:48)
	at org.hibernate.internal.SessionFactoryImpl.close(SessionFactoryImpl.java:928)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:529)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:750)
	at jdk.proxy2/jdk.proxy2.$Proxy110.close(Unknown Source)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.destroy(AbstractEntityManagerFactoryBean.java:662)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:211)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:735)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:692)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:1401)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:651)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:1394)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1219)
	at org.springframework.context.support.AbstractApplicationContext.doClose(AbstractApplicationContext.java:1180)
	at org.springframework.context.support.AbstractApplicationContext.close(AbstractApplicationContext.java:1126)
	at org.springframework.boot.SpringApplicationShutdownHook.closeAndWait(SpringApplicationShutdownHook.java:147)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at org.springframework.boot.SpringApplicationShutdownHook.run(SpringApplicationShutdownHook.java:116)
	at java.base/java.lang.Thread.run(Thread.java:1575)
2025-03-03 20:55:27.396511-05:00 database: close
org.h2.message.DbException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.close(Store.java:374)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1292)
	at org.h2.engine.Database.closeImpl(Database.java:1254)
	at org.h2.engine.Database.close(Database.java:1173)
	at org.h2.engine.Database.onShutdown(Database.java:1158)
	at org.h2.engine.OnExitDatabaseCloser.onShutdown(OnExitDatabaseCloser.java:85)
	at org.h2.engine.OnExitDatabaseCloser.run(OnExitDatabaseCloser.java:114)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 8 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:926)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:463)
	at org.h2.mvstore.RandomAccessStore.compactStore(RandomAccessStore.java:443)
	at org.h2.mvstore.FileStore.compactStore(FileStore.java:890)
	at org.h2.mvstore.FileStore.stop(FileStore.java:275)
	at org.h2.mvstore.MVStore.closeStore(MVStore.java:693)
	at org.h2.mvstore.MVStore.close(MVStore.java:643)
	at org.h2.mvstore.db.Store.close(Store.java:364)
	... 6 more
Caused by: java.lang.AssertionError
	at org.h2.mvstore.RandomAccessStore.moveChunkInside(RandomAccessStore.java:630)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:569)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:481)
	at org.h2.mvstore.RandomAccessStore.lambda$compactMoveChunks$1(RandomAccessStore.java:468)
	at org.h2.mvstore.FileStore.executeFileStoreOperation(FileStore.java:1893)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:922)
	... 13 more
2025-03-04 14:52:28.153863-05:00 database: close
org.h2.message.DbException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.close(Store.java:374)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1292)
	at org.h2.engine.Database.closeImpl(Database.java:1254)
	at org.h2.engine.Database.close(Database.java:1173)
	at org.h2.engine.Database.onShutdown(Database.java:1158)
	at org.h2.engine.OnExitDatabaseCloser.onShutdown(OnExitDatabaseCloser.java:85)
	at org.h2.engine.OnExitDatabaseCloser.run(OnExitDatabaseCloser.java:114)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 8 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:926)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:463)
	at org.h2.mvstore.RandomAccessStore.compactStore(RandomAccessStore.java:443)
	at org.h2.mvstore.FileStore.compactStore(FileStore.java:890)
	at org.h2.mvstore.FileStore.stop(FileStore.java:275)
	at org.h2.mvstore.MVStore.closeStore(MVStore.java:693)
	at org.h2.mvstore.MVStore.close(MVStore.java:643)
	at org.h2.mvstore.db.Store.close(Store.java:364)
	... 6 more
Caused by: java.lang.AssertionError
	at org.h2.mvstore.RandomAccessStore.moveChunkInside(RandomAccessStore.java:630)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:569)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:481)
	at org.h2.mvstore.RandomAccessStore.lambda$compactMoveChunks$1(RandomAccessStore.java:468)
	at org.h2.mvstore.FileStore.executeFileStoreOperation(FileStore.java:1893)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:922)
	... 13 more
2025-03-04 14:59:47.307905-05:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT c.id, c.nombre, c.apellido, a.nombre\000d\000aFROM CLIENT_TABLE c\000d\000a\000d\000a\000d\000a[*]select * from pet_table"; SQL statement:
SELECT c.id, c.nombre, c.apellido, a.nombre
FROM CLIENT_TABLE c


select * from pet_table [42000-232]
2025-03-04 14:59:49.492928-05:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PET_TABLE" not found; SQL statement:
select * from pet_table [42102-232]
2025-03-04 14:59:56.944945-05:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PET_TABLE" not found; SQL statement:
select * from pet_table [42102-232]
2025-03-04 15:31:00.818027-05:00 jdbc[15]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-232]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:217)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:134)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:137)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:360)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:202)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:461)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:550)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:98)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:111)
	at org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.getConnection(DatasourceConnectionProviderImpl.java:126)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcEnvironmentInitiator.java:467)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcIsolationDelegate.delegateWork(JdbcIsolationDelegate.java:61)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.getJdbcEnvironmentUsingJdbcMetadata(JdbcEnvironmentInitiator.java:320)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:129)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:81)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:130)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:263)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:238)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.boot.model.relational.Database.<init>(Database.java:45)
	at org.hibernate.boot.internal.InFlightMetadataCollectorImpl.getDatabase(InFlightMetadataCollectorImpl.java:226)
	at org.hibernate.boot.internal.InFlightMetadataCollectorImpl.<init>(InFlightMetadataCollectorImpl.java:194)
	at org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:171)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:1442)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1513)
	at org.springframework.orm.jpa.vendor.SpringHibernateJpaPersistenceProvider.createContainerEntityManagerFactory(SpringHibernateJpaPersistenceProvider.java:66)
	at org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean.createNativeEntityManagerFactory(LocalContainerEntityManagerFactoryBean.java:390)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:419)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.afterPropertiesSet(AbstractEntityManagerFactoryBean.java:400)
	at org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean.afterPropertiesSet(LocalContainerEntityManagerFactoryBean.java:366)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1859)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1808)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:601)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:523)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:346)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:207)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:970)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:627)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:439)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:318)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1361)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1350)
	at com.example.vetproject.VetprojectApplication.main(VetprojectApplication.java:10)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.boot.devtools.restart.RestartLauncher.run(RestartLauncher.java:50)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 56 more
2025-03-04 15:31:00.862885-05:00 jdbc[15]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "HTE_CLIENT_TABLE" already exists; SQL statement:
create global temporary table HTE_client_table(rn_ integer not null, id bigint, apellido varchar(255), contrasena varchar(255), email varchar(255), nombre varchar(255), telefono varchar(255), usuario varchar(255), primary key (rn_)) TRANSACTIONAL [42101-232]
2025-03-04 15:31:00.864087-05:00 jdbc[15]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "HTE_PETS_TABLE" already exists; SQL statement:
create global temporary table HTE_pets_table(edad integer, rn_ integer not null, cliente_id bigint, id bigint, imagen_url varchar(255), nombre varchar(255), raza varchar(255), tipo varchar(255), primary key (rn_)) TRANSACTIONAL [42101-232]
2025-03-09 21:29:13.586805-05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1423)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:463)
	at com.zaxxer.hikari.pool.HikariProxyConnection.getAutoCommit(HikariProxyConnection.java)
	at org.hibernate.resource.transaction.backend.jdbc.internal.DdlTransactionIsolatorNonJtaImpl.getIsolatedConnection(DdlTransactionIsolatorNonJtaImpl.java:49)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.getIsolatedConnection(GenerationTargetToDatabase.java:60)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.jdbcStatement(GenerationTargetToDatabase.java:112)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:79)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl$DelayedDropActionImpl.perform(SchemaDropperImpl.java:628)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryClosing(SessionFactoryObserverForSchemaExport.java:48)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryClosing(SessionFactoryObserverChain.java:48)
	at org.hibernate.internal.SessionFactoryImpl.close(SessionFactoryImpl.java:928)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:529)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:750)
	at jdk.proxy2/jdk.proxy2.$Proxy113.close(Unknown Source)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.destroy(AbstractEntityManagerFactoryBean.java:662)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:211)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:735)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:692)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:1401)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:651)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:1394)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1219)
	at org.springframework.context.support.AbstractApplicationContext.doClose(AbstractApplicationContext.java:1180)
	at org.springframework.context.support.AbstractApplicationContext.close(AbstractApplicationContext.java:1126)
	at org.springframework.boot.SpringApplicationShutdownHook.closeAndWait(SpringApplicationShutdownHook.java:147)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at org.springframework.boot.SpringApplicationShutdownHook.run(SpringApplicationShutdownHook.java:116)
	at java.base/java.lang.Thread.run(Thread.java:1575)
2025-03-09 21:29:13.597934-05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1423)
	at org.h2.jdbc.JdbcConnection.clearWarnings(JdbcConnection.java:658)
	at com.zaxxer.hikari.pool.ProxyConnection.close(ProxyConnection.java:258)
	at org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.closeConnection(DatasourceConnectionProviderImpl.java:131)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.releaseConnection(JdbcEnvironmentInitiator.java:472)
	at org.hibernate.resource.transaction.backend.jdbc.internal.DdlTransactionIsolatorNonJtaImpl.release(DdlTransactionIsolatorNonJtaImpl.java:106)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.release(GenerationTargetToDatabase.java:134)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl$DelayedDropActionImpl.perform(SchemaDropperImpl.java:639)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryClosing(SessionFactoryObserverForSchemaExport.java:48)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryClosing(SessionFactoryObserverChain.java:48)
	at org.hibernate.internal.SessionFactoryImpl.close(SessionFactoryImpl.java:928)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:529)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:750)
	at jdk.proxy2/jdk.proxy2.$Proxy113.close(Unknown Source)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.destroy(AbstractEntityManagerFactoryBean.java:662)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:211)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:735)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:692)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:1401)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:651)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:1394)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1219)
	at org.springframework.context.support.AbstractApplicationContext.doClose(AbstractApplicationContext.java:1180)
	at org.springframework.context.support.AbstractApplicationContext.close(AbstractApplicationContext.java:1126)
	at org.springframework.boot.SpringApplicationShutdownHook.closeAndWait(SpringApplicationShutdownHook.java:147)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at org.springframework.boot.SpringApplicationShutdownHook.run(SpringApplicationShutdownHook.java:116)
	at java.base/java.lang.Thread.run(Thread.java:1575)
2025-03-09 23:08:40.806665-05:00 database: close
org.h2.message.DbException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.close(Store.java:374)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1292)
	at org.h2.engine.Database.closeImpl(Database.java:1254)
	at org.h2.engine.Database.close(Database.java:1173)
	at org.h2.engine.Database.onShutdown(Database.java:1158)
	at org.h2.engine.OnExitDatabaseCloser.onShutdown(OnExitDatabaseCloser.java:85)
	at org.h2.engine.OnExitDatabaseCloser.run(OnExitDatabaseCloser.java:114)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 8 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:926)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:463)
	at org.h2.mvstore.RandomAccessStore.compactStore(RandomAccessStore.java:443)
	at org.h2.mvstore.FileStore.compactStore(FileStore.java:890)
	at org.h2.mvstore.FileStore.stop(FileStore.java:275)
	at org.h2.mvstore.MVStore.closeStore(MVStore.java:693)
	at org.h2.mvstore.MVStore.close(MVStore.java:643)
	at org.h2.mvstore.db.Store.close(Store.java:364)
	... 6 more
Caused by: java.lang.AssertionError
	at org.h2.mvstore.RandomAccessStore.moveChunkInside(RandomAccessStore.java:630)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:569)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:481)
	at org.h2.mvstore.RandomAccessStore.lambda$compactMoveChunks$1(RandomAccessStore.java:468)
	at org.h2.mvstore.FileStore.executeFileStoreOperation(FileStore.java:1893)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:922)
	... 13 more
2025-03-20 14:13:47.644816-05:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * [*]FROMPETS_TABLE  MED_TABLE"; SQL statement:
SELECT * FROMPETS_TABLE  MED_TABLE [42000-232]
